src/slack/slack.js

```diff
@@ -821,7 +824,7 @@ export class Slack {
 
 
   async addReaction(name, channel, timestamp) {
-    const {ok, error} = await this.apiCall('reactions.add', {name, channel, timestamp}, {passError: true})
+    const {ok, error} = await this.boltApp.client.reactions.add({name, channel, timestamp}, {passError: true})
```

Comment:
do you know if `passError` here works?

Reply:
not sure, removed it.

Reply:
yeah... it shouldn't work, it is an argument to our own `apiCall` function